runner.yml.dist

```diff
@@ -29,6 +29,7 @@ drupal:
     - "./vendor/bin/drush en config_devel -y"
     - "./vendor/bin/drush en oe_subscriptions -y"
     - "./vendor/bin/drush en oe_subscriptions_anonymous -y"
```

Comment:
New version of decoupled auth solves the problem with the install: https://git.drupalcode.org/project/decoupled_auth/-/compare/2.0.0...3.0.1#d18eedf709e2e4ca50781c8fafd2ea1f6e2f5e3d_265_264
The issue is not present anymore